src/app/(docs)/animations/hovers/pulse/page.mdx

```diff
@@ -0,0 +1,37 @@
+import Link from 'next/link'
```

Comment:
can you rebase and add the meta data here like it's added to other hover animations?

Reply:
ok i have rebased but i dont understand the meta part you talked about ,please maybe you could give more context please 🙏

Reply:
add this on top of the file: 

```
export const metadata = {
  title: 'Pulse Hover Effect',
  description: 'A Pulse hover effect animation that gives a playful and energetic effect to your React and Next.js projects, built using Tailwind CSS.',
} 

```

Reply:
ooh thank you very much 🙏😊 i see,let me do that and push asap